pydm/data_plugins/__init__.py

```diff
@@ -41,7 +53,11 @@ def connection_queue(defer_connections=False):
     establish_queued_connections()
 
 
-def establish_queued_connections():
+def establish_queued_connections() -> None:
```

Comment:
These functions that intend to act like queue handlers and modify global state concern me a bit.

Two threads that establish queued connections have the potential to fight each other in unexpected ways (unless you introduce locks, which is less than ideal too). 

It might be wise to switch these to _real_ [queues](https://docs.python.org/3/library/queue.html#queue.Queue)